staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -56,6 +56,14 @@ type debugError interface {
 	DebugError() (msg string, args []interface{})
 }
 
+type ApplyFlags struct {
```

Comment:
This is wrong place for `ApplyFlags`, they should be placed in main apply.go. Compare WaitFlags: https://github.com/kubernetes/kubernetes/blob/d2fd467123aac0060d5cb3fd1a509adabc4ea190/staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go#L76